cli/src/main/scala/scala/scalanative/cli/utils/ConfigConverter.scala

```diff
@@ -83,9 +82,9 @@ object ConfigConverter {
         .withWorkdir(Paths.get(options.config.workdir).toAbsolutePath())
         .withCompilerConfig(nativeConfig)
         .withClassPath(classPath)
-        .withMainClass(main + "$")
+        .withMainClass(main)
```

Comment:
I had to remove "$" because the integration tests would not pass - was this something changed in 0.4.3-SNAPSHOT?

Reply:
Yes, it's one of the breaking changes. In the past, we needed "$" suffix because we were not generating static method forwarders as JVM or Scala.js does, now it is fixed, and we just pass plan `main` without additional suffix